docs/postman/AAS.postman_collection.json

```diff
@@ -171,50 +367,137 @@
+									"type": "string"
+								},
+								{
+									"key": "username",
+									"value": "{{endpoint_ingress_user}}",
+									"type": "string"
+								}
+							]
+						},
+						"method": "DELETE",
+						"header": [
+							{
+								"key": "Content-Type",
+								"type": "text",
+								"value": "application/json"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
 						"url": {
-							"raw": "{{i40-aas-endpoint-registry}}/assetadministrationshells?receiverRole=Operator&semanticProtocol=i40:registry-semanticProtocol/onboarding",
+							"raw": "{{i40-aas-endpoint-registry}}/semanticProtocols/:semanticProtocolId",
 							"host": [
 								"{{i40-aas-endpoint-registry}}"
 							],
 							"path": [
-								"assetadministrationshells"
+								"semanticProtocols",
+								":semanticProtocolId"
 							],
 							"query": [
 								{
-									"key": "receiverRole",
-									"value": "Operator"
+									"key": "REGISTRY_ADMIN_USER",
+									"value": "{{user}}",
+									"disabled": true
 								},
 								{
-									"key": "semanticProtocol",
-									"value": "i40:registry-semanticProtocol/onboarding"
+									"key": "REGISTRY_ADMIN_PASSWORD",
+									"value": "{{pwd}}",
+									"disabled": true
+								}
+							],
+							"variable": [
+								{
+									"key": "semanticProtocolId",
+									"value": "i40:registry-semanticProtocol%2Fonboarding"
 								}
 							]
 						}
 					},
 					"response": []
 				},
 				{
-					"name": "GET All Endpoints",
+					"name": "READ assetadministrationshells",
+					"protocolProfileBehavior": {
+						"disableBodyPruning": true
+					},
 					"request": {
+						"auth": {
+							"type": "basic",
+							"basic": [
+								{
+									"key": "password",
+									"value": "{{endpoint_ingress_pwd}}",
+									"type": "string"
+								},
+								{
+									"key": "username",
+									"value": "{{endpoint_ingress_user}}",
+									"type": "string"
+								}
+							]
+						},
 						"method": "GET",
-						"header": [],
+						"header": [
+							{
+								"key": "Content-Type",
+								"type": "text",
+								"value": "application/json"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
 						"url": {
-							"raw": "{{i40-aas-endpoint-registry}}/listallendpoints",
+							"raw": "{{i40-aas-endpoint-registry}}/AASDescriptors/:aasId",
 							"host": [
 								"{{i40-aas-endpoint-registry}}"
 							],
 							"path": [
-								"listallendpoints"
+								"AASDescriptors",
+								":aasId"
+							],
+							"query": [
+								{
+									"key": "REGISTRY_ADMIN_USER",
+									"value": "{{user}}",
+									"disabled": true
+								},
+								{
+									"key": "REGISTRY_ADMIN_PASSWORD",
+									"value": "{{pwd}}",
+									"disabled": true
+								}
+							],
+							"variable": [
+								{
+									"key": "aasId",
+									"value": "www.sampleOperator/aas"
```

Comment:
Escape special character "/" here: "www.sampleOperator%2Faas"

Reply:
Right, corrected

Reply:
It's not corrected on the current commit. Did you upload the up-do-date Postman collection?